### DISCUSSION TITLE
[bug] 获取到资源不能播放

### DISCUSSION THREAD

**Discussion by jssda** [1715173329/UnblockNeteaseMusic#682] [Q&A]
# 问题
控制台日志显示已经获取到了连接 ，并且点击浏览器打开也能够播放，网易云这首歌曲直接跳过（没有任何提示），跳到下一首能播放的播放了。
## 环境
- 系统 Debian11
- 网易云版本：1.2.1 
## 日志
![image](https://user-images.githubusercontent.com/33473639/166444360-5d49e08f-8cf2-481b-8ed7-4a4af7c6a177.png)
## 浏览器播放
我拉取到了歌曲中端，没问题可以播放
![image](https://user-images.githubusercontent.com/33473639/166444721-52802774-b3ed-4c12-a304-f48f02b86c25.png)

**Comment by 1715173329** [comment]
证书信任了吗？或者是否开启了跳过证书验证？

**Comment by jssda** [comment]
信任了，代理使用的是https端口8081。
哦对 配置代理的工具是Clash， 不知道有没有影响
![image](https://user-images.githubusercontent.com/33473639/166447031-c6f36ad5-7f3b-4f31-8b4a-399420c89cf5.png)
跳过验证这个东西，应该是没开的吧。。不是直接配上证书就行么， 没配置过其他东西

**Comment by 1715173329** [comment]
需要走 http 代理端口，可能还需要设置 endpoint `-e https://music.163.com`。
没测试过 clash，如果有条件的话可以试试用 iptables 劫持。

**Reply by RWDai** [reply]
一样的情况 加了-e可以用了 请问这个-e参数具体什么含义 替换这个端点host是干嘛用的

**Reply by IT-yao** [reply]
> 一样的情况 加了-e可以用了 请问这个-e参数具体什么含义 替换这个端点host是干嘛用的

l老哥 你咋弄的
